src/bindings/python/src/openvino/tools/mo_lite/subprocess_main.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging as log
+import os
+import subprocess # nosec
+import sys
+
+
+def check_python_version():
+    """
+    Checks python version to be greater or equal than 3.4
+    :return: exit code (1 - error, None - successful)
+    """
+    if sys.version_info < (3, 4):
+        print('Python version should be of version 3.4 or newer')
+        return 1
+
+
+def log_ie_not_found():
+    log.error("Could not find the Inference Engine or nGraph Python API.\n"
+              "Consider building the Inference Engine and nGraph Python APIs"
+              " from sources or try to install OpenVINO (TM) Toolkit using pip \npip install openvino")
+
+
+def log_mo_root_dir_not_found():
+    log.error("Could not find the ModelOptimizer root module directory.\n"
+              "Consider setting PYTHONPATH to the openvino tools folder (usually openvino/tools/mo)")
```

Comment:
`openvino/tools/mo` to mo-lite version

Reply:
Most likely mo-lite will be renamed to mo and the rest of MO to mo_legacy. mo-lite is temporary name.